File: answers/VrishtiSharma/Conditonals/Q1.py
n = int(input("Enter the basic salary : "))
x = input("Enter grade : ")
hra = 20/100 * n
da = 50/100 * n
pf = 11/100 * n
if x == 'A':
    allow = 1700
elif x == 'B':
    allow = 1500
else:
    allow = 1300
totalSalary = n + hra + da + allow - pf
print(int(totalSalary))

File: answers/HarshArya-2107/Conditional/q1.cpp
#include <iostream>
#include <conio.h>
#include <string.h>
using namespace std;

int main() {
    double basic,da,hra,pf,allow,totalSalary;
    char grade;

    cout << "\n Enter basic salary (int) and grade (upper case) : ";
    cin >> basic >> grade;
    da = (double) 50 / 100 * basic;
    hra = (double) 20 / 100 * basic;
    pf = (double) 11 / 100 * basic;

    if(grade == 'A') {
        allow = 1700;
    } else if(grade == 'B') {
        allow = 1500;
    } else {
        allow = 1300;
    }

    totalSalary = basic + hra + da + allow - pf;
    cout << (int) totalSalary;

    return 0;
} 
